components/org.wso2.carbon.identity.application.authenticator.fido2/src/main/java/org/wso2/carbon/identity/application/authenticator/fido2/dto/RegistrationRequest.java

```diff
@@ -31,9 +31,6 @@
 @EqualsAndHashCode(callSuper = false)
 public class RegistrationRequest {
 
-    @JsonProperty("username")
-    String username;
```

Comment:
If this is present in already released version, let's deprecate first and remove in a later version